docs/content/docs/build/containers.md

```diff
@@ -125,6 +125,16 @@ container.on("connected", () => {
     // handle connected
     // enable editing if disabled
 });
+
+user.on("idle", () => {
```

Comment:
I like this example, but I worry that folks might think these events are something the framework provides. Maybe just add a short comment or something? I'm probably being over cautious though.